tests/unit/FormTextArea.spec.js

```diff
@@ -100,26 +112,32 @@ describe('FormTextArea', () => {
     const errorText = 'This field has an error';
     const wrapper = factory({
       name: 'FormTextAreaField',
-      error: errorText,
-      validation: 'required'
     });
-    
+
+    wrapper.setProps({value: "", validation: 'required'});
+    expect(wrapper.find('.is-invalid').exists()).toBe(true);
     expect(wrapper.find('textarea').classes('is-invalid')).toBe(true);
     expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
     expect(wrapper.find('.invalid-feedback').text()).toContain(requiredText);
+
+    wrapper.setProps({value: "text", error: errorText});
+    expect(wrapper.find('.is-invalid').exists()).toBe(true);
+    expect(wrapper.find('textarea').classes('is-invalid')).toBe(true);
+    expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
     expect(wrapper.find('.invalid-feedback').text()).toContain(errorText);
   });
 
   it('removes the validation error messages when the field is valid.', () => {
     const errorText = 'This field has an error';
```

Comment:
This variable is no longer used and should be removed. Which IDE are you using? PHPStorm automatically highlights unused variables; this is a tool that might be worth looking into for your specific IDE.